src/main/java/com/kodilla/ecommercee/domain/Product/Product.java

```diff
@@ -21,10 +21,9 @@
     public Product() {
     }
 
-    public Product(String description, BigDecimal price, Group group) {
+    public Product(String description, BigDecimal price) {
         this.description = description;
         this.price = price;
-        this.group = group;
```

Comment:
wydaje mi się, że brakuje w konstruktorze info o id grupy, by relacja miała po czym łączyć.

Reply:
@naros1 takie rozwiązanie ma sens o ile w klasie Product  podam Group group. 
Szczerze, to nie widze w czym moje rozwiązanie jest gorsze od tego, które zaproponowałeś

Reply:
zastanawiam się tylko, w którym momencie podczas tworzenia produktu powinniśmy podać info o tym do jakiej grupy należy. Obecnie, z tego co widzę, usunąłeś taką informację z konstruktora.

Reply:
> @naros1 takie rozwiązanie ma sens o ile w klasie Product podam Group group.
> Szczerze, to nie widze w czym moje rozwiązanie jest gorsze od tego, które zaproponowałeś

Do którego rozwiązania się odnośisz proponuję jeżeli mowa o List carts = new ArrayList<>(); to chodzi mi dokładnie o miejsce na screenie poniżej 
![komentarzPNG](https://user-images.githubusercontent.com/52628091/71772264-e71fe780-2f48-11ea-97d0-3a2ec3b35988.PNG)

komentarz przy testach wyjaśnia dokładniej dlaczego jest lepsze